services/app-api/forms/mcpar.json

```diff
@@ -2369,7 +2361,7 @@
                   "section": "Section D: Plan-Level Indicators",
                   "subsection": "Topic IV. Appeals, State Fair Hearings & Grievances",
                   "spreadsheet": "D1_Plan_Set",
-                  "alert": "<strong>Beginning June 2025, Indicators D1.IV.1a-c must be completed. Submission of this data before June 2025 is optional; if you choose not to respond prior to June 2025, enter \"N/A\".</strong>",
+                  "alert": "<strong>Beginning June 2025, Indicators D1.IV.1a-c must be completed. Submission of this data before June 2025 is optional; if you choose not to respond prior to June 2025, enter “N/A”.</strong>",
```

Comment:
would this case be "NR"?

Reply:
No, in this case we're still using "N/A". I think eventually the BOs would like "NR" as an option throughout the MCPAR, but it's explicitly an option for Prior Authorization for now.